Programs/USFWS/Program.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Program xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../Program.xsd">
+
+  <Name>Upper Klamath - USFWS</Name>
+  <MachineName>USFWS</MachineName>
```

Comment:
Bit concerned that USFWS is a national outfit while this program is specific to the Klamath. How about:

- `UKUSFWS` or is that confusing with England?
- `Klamath`
- `KUSFWS`